test/Microsoft.Health.Fhir.Shared.Tests.Integration/Features/Operations/Reindex/ReindexJobTests.cs

```diff
@@ -902,14 +902,21 @@ private ResourceWrapper CreateObservationResourceWrapper(string observationId)
             return wrapper;
         }
 
-        private ResourceWrapper CreateSearchParamResourceWrapper(SearchParameter searchParam, bool deleted = false)
+        private ResourceWrapper CreateSearchParamResourceWrapper(SearchParameter searchParam, string id = "searchParam1", bool deleted = false)
```

Comment:
You've added this new id parameter, but it looks like it isn't used and the id is still hardcoded to searchParam1.

Reply:
It is true, I am only using it on line 570, as only this test case seemed troubled by the same param id.

Reply:
But this parameter isn't used in the method anywhere. You set it on line 570, but that value is never used.

Reply:
You are correct! Sorry.  Removed.